### DISCUSSION TITLE
I sent a photo of a student ID with my face in one photo, but it gives an error related to payment information, although everything is specified exactly.

### DISCUSSION THREAD

**Discussion by Ap1kIvanich** [community/community#115461] [GitHub Education]
### Hi everyone,

I sent a photo of a student card with my face in one photo, but it gives an error related to payment information, although everything is specified exactly.

**Comment by samus-aran** [comment]
Hi @Ap1kIvanich 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your billing information. Billing and your translated evidence should match.
- Ensure your evidence correspond to details in your GitHub billing info. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
